src/collector/config/m3collector.yml

```diff
@@ -0,0 +1,87 @@
+listenAddress:
+  type: config
+  value: 0.0.0.0:7206
+
+metrics:
+  scope:
+    prefix: collector
+  prometheus:
+    handlerPath: /metrics
+    listenAddress: 0.0.0.0:7207 # until https://github.com/m3db/m3/issues/682 is resolved
+  sanitization: prometheus
+  samplingRate: 1.0
+  extended: none
+
+etcd:
+  env: default_env
+  zone: embedded
+  service: m3collector
+  cacheDir: /var/lib/m3kv
+  etcdClusters:
+    - zone: embedded
+      endpoints:
+        - m3db_seed:2379
+
+reporter:
+  cache:
+    capacity: 200000
+    freshDuration: 5m
+    stutterDuration: 1m
+
+  matcher:
+    initWatchTimeout: 10s
+    rulesKVConfig:
+      namespace: /r2/rules
+    namespacesKey: namespaces
+    ruleSetKeyFmt: rulesets/%s
+    namespaceTag: application
+    defaultNamespace: global
+    nameTagKey: __name__
+    matchRangePast: 2m
+    sortedTagIteratorPool:
+      size: 8192
+      watermark:
+        low: 0.7
+        high: 1.0
+
+  client:
+    placementKV:
+      namespace: /r2/m3aggregator
+    placementWatcher:
+      key: placement
```

Comment:
you're watching for m3agg placements right? I think this will be the name of the service in the placement so make this `m3agg` maybe?